services-js/registry-certs/client/birth/components/DisclosureComponent.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react';
+
+import { css } from 'emotion';
+
+import {
+  CLEARDEFAULTSTYLING,
+  OPTIMISTIC_BLUE_DARK,
+} from '@cityofboston/react-fleet';
+
+export default function DisclosureComponent(): JSX.Element {
```

Comment:
This should probably be named more specifically to apostille messaging?

Reply:
Duh - thx!